vqpy/__init__.py

```diff
@@ -62,7 +62,7 @@ def launch(cls_name,
         outputs = detector.inference(frame_image)
         tracked_tracks, _, frame = tracker.update(outputs, frame)
         for task in tasks:
-            task.vqpy_update(frame_id, tracked_tracks)
+            task.vqpy_update(frame_id, tracked_tracks, frame)
```

Comment:
Do we need both `tracked_tracks` and `frame`? Or just `frame`

Reply:
`frame` is sufficient, removed `frame_id` and `tracked_tracks`.